pkg/plugins/builtin/email/email.go

```diff
@@ -70,38 +80,43 @@ func validConfig(config interface{}) error {
 func exec(stepName string, config interface{}, ctx interface{}) (interface{}, interface{}, error) {
 	cfg := config.(*Config)
 
-	parameters := struct {
-		From    string `json:"from"`
-		To      string `json:"to"`
-		Subject string `json:"subject"`
-		Body    string `json:"body"`
-	}{
-		cfg.From,
-		strings.Join(cfg.To, ","),
-		mime.BEncoding.Encode("UTF-8", cfg.Subject),
-		cfg.Body,
-	}
+	message := mail.NewMessage()
 
-	buffer := new(bytes.Buffer)
+	recipients := make([]string, len(cfg.To))
+	for i, recipient := range cfg.To {
+		recipients[i] = message.FormatAddress(recipient, "")
```

Comment:
What is the goal of calling ```message.FormatAddress``` ?

https://github.com/go-mail/mail/blob/v2.3.1/message.go#L135

if name is empty, it returns the address :)

Reply:
thanks!